app/views/search/_delay_modal.html.erb

```diff
@@ -0,0 +1,24 @@
+<!--begin modal-->
+<div class="modal" id="delayWarningModal" tabindex="-1" role="dialog" aria-labelledby="delayWarningModal">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h4 class="modal-title" id="delayWarningModalLabel">Possible Delay Ahead</h4>
+      </div>
+      <div class="modal-body">
+        <p>
+          This collection is known to cause delays and may take longer to download than expected.
+        </p>
+        <p>
+          <span id='delayOptionalMessage'></span>
+        </p>
+      </div>
+      <div class="modal-footer">
+        <a class="button text-button" id="delayOk" data-dismiss="modal" href="#">Okay</a>
```

Comment:
I'm not sure if there's a UX precedent for this already in EDSC, but I think there may be a more effective way to word this button than "Okay" - I clicked on this button and my instinctual expectation for the "Okay"/"OK" button is that the modal would just disappear, as if I was just "dismissing" the modal or saying "okay." 

Of course, my expectation as a user should have been that this button brings me on to the download - a more "action specific" button might feel more natural (maybe "Continue" or "Download"). A counterargument I could understand is that any regular user may naturally understand "Okay" to continue them through the workflow (because what good would hitting "Okay" be otherwise?).

It's possible I'm alone in my initial perception of this UX flow, so please feel free to disagree with me since I can understand a solid argument for the "Okay" button!

Reply:
@jennywagg Seems  like a valid point to me!  'Okay' is also a bit informal.

Perhaps the button could say 'Proceed' - would that be more appropriate?  Open to other suggestions as well.

Reply:
"Continue" is better